onnx/common/ir.h

```diff
@@ -950,6 +953,17 @@ friend struct Value;
     initializers_.push_back(std::move(initializer));
     initializer_names_.push_back(std::move(name));
   }
+  // For IR >= 4, initializer is not required to exist in input
+  // Add initializer into initializer node list and return its Value
+  Value* addInitializerNode(Tensor initializer, std::string name) {
+    Value* init_value = initializerNode_->addOutput();
+    std::vector<Dimension> dim_sizes{initializer.sizes().cbegin(),
+                                    initializer.sizes().cend()};
+    init_value->setUniqueName(name);
+    init_value->setSizes(dim_sizes);
+    init_value->setElemType(initializer.elem_type());
+    return init_value;
+  }
   void eraseInitializer(const std::string &name) {
```

Comment:
Need to erase created init node here... Doing

Reply:
Check eraseInitializerAndInput